src/commcare_cloud/ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```
@@ -81,14 +97,35 @@ cloud:
 {% endif %}
 
 # Only allow indices to be created by an explicit "create" command
+{% if elasticsearch_version == '2.4.6'%}
 action.auto_create_index: false
+{% endif %}
+{% if elasticsearch_version == '7.9.1' %}
+action.auto_create_index: .watches,.triggered_watches,.watcher-history-*
+{% endif %}
 
 
 {% if es_local_repo and not backup_es_s3%}
 path.repo: ["{{ elasticsearch_data_dir }}/snapshot"]
 {% endif %}
 
 
+
+{% if elasticsearch_version == '7.9.1' %}
+cluster.initial_master_nodes:
+{% for item in groups.elasticsearch %}
+{% if hostvars[item].elasticsearch_master is defined %}
```

I think you want this to also be True correct:
```suggestion
{% if hostvars[item].elasticsearch_master|default(False) %}
```